pytest-embedded-serial/pytest_embedded_serial/serial.py

```diff
@@ -47,13 +47,16 @@ def __init__(
         port_location: Optional[str] = None,
         baud: int = DEFAULT_BAUDRATE,
         meta: Optional[Meta] = None,
+        stop_after_init: bool = False,
+        ports_to_occupy: List[str] = (),
```

Comment:
should be put in all the subclasses to filter these ports out while auto-detecting...

Reply:
Sure, thank you!

Reply:
@hfudev  I have added it to EspSerial. ArduinoDut and IdfDut don't need it.